### PR TITLE
[A11y] Make the project options panels accessible

### PR DESCRIPTION
**Pull request by iainx**
Adds label and help to most of the project options panels in the main repo. Panels using Xwt can't be
made accessible yet. Partially fixes #53537

### CODE REVIEW

**Comment by iainx** [comment]
@mrward can you check the descriptions of the NuGet package options actually reflect what the widget does?

File: main/src/addins/MonoDevelop.Packaging/MonoDevelop.Packaging.Gui/GtkNuGetPackageMetadataOptionsPanelWidget.cs [hunk 1]
```diff
@@ -46,6 +48,91 @@ public GtkNuGetPackageMetadataOptionsPanelWidget ()
...
+			                                                           GettextCatalog.GetString ("Project URL"),
+			                                                           GettextCatalog.GetString ("Enter the project URL for the NuGet package"));
+			packageProjectUrlTextBox.SetAccessibilityLabelRelationship (packageProjectUrlLabel);
+
+			packageIconUrlTextBox.SetCommonAccessibilityAttributes ("NugetMetadata.Icon",
+			                                                        GettextCatalog.GetString ("Icon URL"),
+			                                                        GettextCatalog.GetString ("Enter the URL for the NuGet package's icon"));
+			packageIconUrlTextBox.SetAccessibilityLabelRelationship (packageIconUrlLabel);
+
+			packageLicenseUrlTextBox.SetCommonAccessibilityAttributes ("NugetMetadata.license",
```
Comment (Therzok) [inline main/src/addins/MonoDevelop.Packaging/MonoDevelop.Packaging.Gui/GtkNuGetPackageMetadataOptionsPanelWidget.cs]: ~~license~~ License?

File: main/src/addins/MonoDevelop.Packaging/MonoDevelop.Packaging.Gui/GtkNuGetPackageMetadataOptionsPanelWidget.cs [hunk 2]
```diff
@@ -46,6 +48,91 @@ public GtkNuGetPackageMetadataOptionsPanelWidget ()
...
+			                                                           GettextCatalog.GetString ("License URL"),
+			                                                           GettextCatalog.GetString ("Enter the URL for the NuGet package's license"));
+			packageLicenseUrlTextBox.SetAccessibilityLabelRelationship (packageLicenseUrlLabel);
+
+			packageRequireLicenseAcceptanceCheckBox.SetCommonAccessibilityAttributes ("NugetMetadata.acceptance",
```
Comment (Therzok) [inline main/src/addins/MonoDevelop.Packaging/MonoDevelop.Packaging.Gui/GtkNuGetPackageMetadataOptionsPanelWidget.cs]: ~~acceptance~~ Acceptance?

File: main/src/addins/MonoDevelop.Packaging/MonoDevelop.Packaging.Gui/GtkNuGetPackageMetadataOptionsPanelWidget.cs [hunk 3]
```diff
@@ -46,6 +48,91 @@ public GtkNuGetPackageMetadataOptionsPanelWidget ()
...
+			                                                                          GettextCatalog.GetString ("Require License Acceptance"),
+			                                                                          GettextCatalog.GetString ("Check to require the user to accept the NuGet package's license"));
+			packageRequireLicenseAcceptanceCheckBox.SetAccessibilityLabelRelationship (packageRequireLicenseAcceptanceLabel);
+
+			packageDevelopmentDependencyCheckBox.SetCommonAccessibilityAttributes ("NugetMetadata.development",
```
Comment (Therzok) [inline main/src/addins/MonoDevelop.Packaging/MonoDevelop.Packaging.Gui/GtkNuGetPackageMetadataOptionsPanelWidget.cs]: ~~development~~ Development?

File: main/src/addins/MonoDevelop.Packaging/MonoDevelop.Packaging.Gui/GtkNuGetPackageMetadataOptionsPanelWidget.cs [hunk 4]
```diff
@@ -46,6 +48,91 @@ public GtkNuGetPackageMetadataOptionsPanelWidget ()
...
+			                                                                       GettextCatalog.GetString ("Development Dependency"),
+			                                                                       GettextCatalog.GetString ("Check to indicate that this is a development dependency"));
+			packageDevelopmentDependencyCheckBox.SetAccessibilityLabelRelationship (packageDevelopmentDependencyLabel);
+
+			packageTagsTextBox.SetCommonAccessibilityAttributes ("NugetMetadata.tags",
```
Comment (Therzok) [inline main/src/addins/MonoDevelop.Packaging/MonoDevelop.Packaging.Gui/GtkNuGetPackageMetadataOptionsPanelWidget.cs]: ~~tags~~ Tags?

File: main/src/core/MonoDevelop.Ide/MonoDevelop.Ide.Gui.Pads/ErrorListPad.cs [hunk 5]
```diff
@@ -904,7 +902,7 @@ void AddTaskInternal (TaskListEntry t)
 				case TaskSeverity.Warning:
 					stock = iconWarning;
 					warningCount++;
-					UpdateWarningsNum ();	
+					UpdateWarningsNum ();
 					break;
 				default:
 					stock = iconInfo;
```
Comment (Therzok) [inline main/src/core/MonoDevelop.Ide/MonoDevelop.Ide.Gui.Pads/ErrorListPad.cs]: This file has been reformatted.

File: main/src/addins/MonoDevelop.Packaging/MonoDevelop.Packaging.Gui/GtkNuGetPackageMetadataOptionsPanelWidget.cs (hunk 1 above)
Comment (mrward) [inline main/src/addins/MonoDevelop.Packaging/MonoDevelop.Packaging.Gui/GtkNuGetPackageMetadataOptionsPanelWidget.cs]: Capital 'G' in NuGet too :)

**Pull request merged**